src/Resources/dto/types.ts

```diff
@@ -0,0 +1,104 @@
+type Week = {
+    id: number,
+    year: number,
+    calenderWeek: number,
+    days: Array<Day>,
+};
+
+type Transaction = {
+    data: [{
+        type: string,
+        date: string,
+        description: string,
+        amount: number,
+    }],
+    difference: number,
+};
+
+type Day = {
+    id: number | null,
+    dateTime: Date,
+    week: Week,
+    meals: Array<Meal>,
+    lockParticipationDateTime: Date,
+};
+
+type Meal = {
+    id: number | null,
+    dish: Dish,
+    price: number,
+    participationLimit: number,
+    day: Day,
+    dateTime: Date,
+    participants: Array<Participant>,
+};
+
+type Dish = {
+    id: number,
+    slug: string,
+    title_en: string,
+    title_de: string,
+    description_en: string | null,
+    description_de: string | null,
+    category: Category | null,
+    price: number,
+    enabled: boolean,
+    currentLocale: string,
+    variations: Array<DishVariation>,
+};
+
+type DishVariation = Dish;
```

Comment:
What's the difference between Dish and DishVariation here?
A Dish-Variation has a parent dish. Don't we need it here?